Pipfile

```diff
@@ -9,6 +9,8 @@ thoth-common = "*"
 thoth-python = "*"
 thoth-analyzer = "*"
 thoth-storages = "*"
+thoth-messaging = "*"
```

Comment:
We can remove thoth-messaging as the message will be sent by another task

Reply:
My only worry is that some topic name changes, which shouldn't happen.  If we are relatively sure this won't happen then I am fine removing it

Reply:
then let's leave it!! Thanks Kevin!